WordPress/openverse
# Update sample data to remove invalid licenses

## Fixes



Follow up to #3440

## Description



#3440 stopped Flickr from attempting to record images with invalid licenses. We have some of those licenses in our sample data. This PR just does a replace of all of the following:

`https://creativecommons.org/licenses/by-nd-nc/2.0/jp/`

with

`https://creativecommons.org/licenses/by-nc-nd/2.0/'

See #3440 for more context.

## Testing Instructions


Checkout this branch. Run `just catalog/pgcli` to go check the licenses shown locally for flickr:

```
openledger> select distinct meta_data->>'license_url' from image where provider='flickr';
+-------------------------------------------------------+
| ?column?                                              |
|-------------------------------------------------------|
| https://creativecommons.org/licenses/by-nc/2.0/       |
| https://creativecommons.org/licenses/by-nd-nc/2.0/jp/ |
| https://creativecommons.org/publicdomain/mark/1.0/    |
| https://creativecommons.org/licenses/by-nc-sa/2.0/    |
| https://creativecommons.org/licenses/by-sa/2.0/       |
| https://creativecommons.org/licenses/by-nd/2.0/       |
| https://creativecommons.org/publicdomain/zero/1.0/    |
| https://creativecommons.org/licenses/by/2.0/          |
+-------------------------------------------------------+
```

Note the invalid license.

Run `just api/init` and ensure the sample data can still be loaded.

Verify the invalid license has been replaced:
```
openledger> select distinct meta_data->>'license_url' from image where provider='flickr';
+-------------------------------------------------------+
| ?column?                                              |
|-------------------------------------------------------|
| https://creativecommons.org/licenses/by-nc/2.0/       |
| https://creativecommons.org/publicdomain/mark/1.0/    |
| https://creativecommons.org/licenses/by-nc-sa/2.0/    |
| https://creativecommons.org/licenses/by-sa/2.0/       |
| https://creativecommons.org/licenses/by-nd/2.0/       |
| https://creativecommons.org/publicdomain/zero/1.0/    |
| https://creativecommons.org/licenses/by/2.0/          |
| https://creativecommons.org/licenses/by-nc-nd/2.0/ |
+-------------------------------------------------------+
```


## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
